lib/spack/spack/binary_distribution.py

```diff
@@ -680,8 +684,18 @@ def get_buildfile_manifest(spec):
                     added = True
 
             if relocate.needs_text_relocation(m_type, m_subtype):
-                data["text_to_relocate"].append(rel_path_name)
-                added = True
+                contents = ""
+                with io.open(path_name, "r", errors="ignore") as f:
```

Comment:
You should open in binary mode (`rb`), otherwise you have to deal with (broken) text encodings.

Reply:
Done